sass/tji/_helpers.scss

```diff
@@ -1,5 +1,6 @@
-.record-count {
-    font-size: 2em;
+.tji-record-count {
```

Comment:
general style comments -- 
1) could we stick to variables so we don't have 843587345 variations of colors, font-sizes, padding, margin, etc.  If you pull in master you'll see color and size defs in sass/tji/_variable.scss. 
2) could you stick to the BEM convention for classnames to keep our styles scoped, etc. Perks of BEM [here](https://css-tricks.com/bem-101/)